lib/gui/models/store.js

```diff
@@ -189,6 +189,16 @@ const storeReducer = (state = DEFAULT_STATE, action) => {
         throw errors.createError(`Invalid state percentage: ${action.data.percentage}`);
       }
 
+      const HUNDRED = 100;
+      if (action.data.percentage > HUNDRED) {
+        throw errors.createError(`Excessive state percentage: ${action.data.percentage}`);
+      }
+
+      const ZERO = 0;
+      if (action.data.percentage < ZERO) {
+        throw errors.createError(`Deficient state percentage: ${action.data.percentage}`);
```

Comment:
That's true, it should be overt from the value being shown which predicate was triggered. I'll do this.